### ISSUE TITLE
No eth0

### ISSUE THREAD

**Issue by AndrewCapon** [polarfire-soc/polarfire-soc-discovery-kit-reference-design#2]
Hi Guys,

Should ethernet be enabled?

I do not see an eth0:

```
root@mpfs-disco-kit:~# ifconfig
lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 89  bytes 7682 (7.5 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 89  bytes 7682 (7.5 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

Thanks

Andy

**Comment by hughbreslin** [comment]
Hi Andy,

sorry for the delayed response - which Linux image did you grab? Can you please try the one included in the Yocto [2024.02.1 release](https://github.com/polarfire-soc/meta-polarfire-soc-yocto-bsp/releases/tag/v2024.02.1)

**Comment by AndrewCapon** [comment]
Hi @hughbreslin,

I just built my own with it enabled the end.

I had tried both the images available at the time, one being the one you linked, neither had eth0:

https://github.com/polarfire-soc/meta-polarfire-soc-yocto-bsp/releases/tag/v2024.02.1 
https://github.com/polarfire-soc/polarfire-soc-discovery-kit-reference-design/releases/tag/2024.04

**Comment by hughbreslin** [comment]
Hi Andi, sorry for the delay - I've retested the images have we online with the reference design and I can see eth0 in both of them. Are you saying its working with a custom build and not with the provided images?

**Comment by AndrewCapon** [comment]
Hi Hugh,

That's weird, they didn't work here! It of course may have been user error.

I just had a look to find out what I did to get it working here as I can't remember but it looks like I might have deleted the yocto build! The one I built was a bit unstable to say the least!

When I get back to looking at the PolarFire again I can add some more info here, I will close this now though and then reopen it if needed.

Cheers

Andy
